code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -642,6 +642,30 @@
 		playsound(M, "sparks", 50, 1)
 	..()
 
+/datum/reagent/toxin/wasting_toxin //Used by the changeling death sting. Very weak, but never metabolizes and will kill the target unless they receive treatment
+	name = "Wasting Toxin"
+	id = "wasting_toxin"
+	description = "An insidious, biologically-produced poison. The body is incapable of metabolizing it, meaning it will slowly kill them unless help is received."
+	metabolization_rate = 0
+	color = rgb(51, 202, 63)
+	toxpwr = 0.5
```

Comment:
It might be intended to be very slow but this risk becoming useless if the victim has a healing virus.